submissions/Esebame1/Week_1_solution.py

```diff
@@ -0,0 +1,11 @@
+for a in range (1,100):
+  if a%3==0 and a%5==0:
+    print('Fizzbuzz\n')
+     continue
+  elif a%3==0:
+    print('Fizz\n')
+     continue
+  elif a%5==0:
+    print('Buzz\n')
+     continue
+  print (str(a)+'\n')
```

Comment:
It's safer to add else before the last print statement so this statement only run when all other 
`if else` check fails. With the `else`, you won't need to add `continue` in each block for this to work as expected.

Reply:
Thanks, I will do that